x-pack/plugins/security/server/config.ts

```diff
@@ -225,25 +234,19 @@ export function createConfig(
   const sortedProviders: Array<{
```

Comment:
__note:__ since we're adding more common config properties the amount of info we're duplicating in `sortedProviders` is increasing, so I decided to take a step back and keep `authc.providers` as the only source of truth.